testing/testing.py

```diff
@@ -0,0 +1,9 @@
+import json
+from ocr import ReceiptOcr
+
+if __name__ == '__main__':
```

Comment:
you can remove this line and test by using "python testing.py" instead. I think it'll be the better way since Heroku might be confused as to which is the main module. Maybe, LOL. But just in case lets get this line deleted and de-indent the following lines.

Reply:
Line 4